vars/withAvd.groovy

```diff
@@ -70,7 +70,7 @@ private def launchAvd(String avdName, boolean headless) {
 
 private def firstAvailablePortInRange(start, end) {
     return sh(
-            script: "(netstat -atn | awk '{printf \"%s\\n%s\\n\", \$4, \$4}' | grep -oE '[0-9]*\$'; seq ${start} ${end}) | sort -n | uniq -u | head -n 1",
+            script: "(ss -atn | awk '{printf \"%s\\n%s\\n\", \$4, \$4}' | grep -oE '[0-9]*\$'; seq ${start} ${end}) | sort -n | uniq -u | head -n 1",
```

Comment:
Thanks for PR!

Unfortunately we use these steps primarily on macOS where `ss` isn't installed by default, could we perhaps do the following instead and check for the presence of `netstat` and fallback to `ss` if not available?

```
def netstatAvailable = sh(script: "command -v netstat", returnStdout: true)
def netstatCommand = if (netstatAvailable == 0) "netstat" else "ss"
return sh(
    script: "(${netstatCommand} -atn | awk '{printf \"%s\\n%s\\n\", \$4, \$4}' | grep -oE '[0-9]*\$'; seq ${start} ${end}) | sort -n | uniq -u | head -n 1",
    returnStdout: true
).trim()
```